builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -108,6 +108,40 @@ def loadServiceConfigurationData() {
           updateConfigValue("{apigee_password}", PASS)
         }
       }
+
+      if (config_loader.AZURE) {
```

Comment:
Since azure is an extension that can be enabled/disabled by admin, we'll need the check below wherever we use config_loader.AZURE in these build packs - 
`if (config_loader.AZURE && config_loader.AZURE.IS_ENABLED)
`
Please see https://github.com/tmobile/jazz-installer/pull/490#discussion_r268013956

Reply:
Let me fix this

Reply:
Done on this

Reply:
Given another thought on this, we should not have the IS_ENABLED check over here, unless you consider building a logic in "admin" that will trigger redeploying jazz_metrics after the extension is enabled